src/js/menu/menu-item.js

```diff
@@ -17,7 +17,18 @@ class MenuItem extends Button {
 
   constructor(player, options) {
     super(player, options);
+
+    this.selectable = options['selectable'];
+
     this.selected(options['selected']);
+
+    if (this.selectable) {
+      // TODO: May need to be either menuitemcheckbox or menuitemradio,
+      //       and may need logical grouping of menu items.
+      this.el_.setAttribute('role', 'menuitemcheckbox');
```

Comment:
Don't base your understanding of whether accessibility is working on ChromeVox!!! You'll hurt your credibility with accessibility experts ;-) It's convenient because it's free, but Firefox+NVDA on a PC is a much better free, standards-compliant alternative. See http://webaim.org/projects/screenreadersurvey/.

That said, I've also been looking at some options around `listbox` and `option`. Semantically, the menus (e.g. the CC menu) are complicated because they have a current state (selected item), potentially a non-actionable title, a settings menu option (which is not "selectable", but is actionable), and a set of options of which one can be picked (we had that discussion about whether more than one could be picked, if we combine CC and Subtitles into a single menu, but I think we concluded they couldn't).